### ISSUE TITLE
Babel Preset Query

### ISSUE THREAD

**Issue by ccorcos** [deepsweet/babel-istanbul-loader#6]
I need to use the babel preset query to resolve the paths to the preset files because my build configurations are in a sibling directory to my application

For example, I'm using this setup for babel:

``` js
        {
          test: /\.js$/,
          loader: 'babel',
          exclude: /node_modules/,
          query: {
            presets: [
              'babel-preset-es2015',
              'babel-preset-react',
              'babel-preset-stage-0',
            ].map(require.resolve),
          },
        },
```

Babelrc doesn't resolve correctly -- any ideas how we could fix this and make the presets a query option?

**Comment by ccorcos** [comment]
https://github.com/babel/babel-loader/blob/master/index.js#L36

**Comment by bbottema** [comment]
Indeed, I would like similar support for this to the way it works with the `babel-loader`, ie:

`loader: 'babel-istanbul?presets[]=es2015&cacheDirectory'`
